src/pages/api/bookmark/index.page.ts

```diff
@@ -1,27 +1,39 @@
 import { CV } from 'models/cv'
 import { NextApiRequest, NextApiResponse } from 'next'
-import bookmarkProject from 'utils/db/firebase/queries/bookmarkProject'
+import {
+  bookmarkProject,
+  unbookmarkProject,
+} from 'utils/db/firebase/queries/bookmarkProject'
 
 export interface BookmarkProjectData {
   address: string
   projectId: number
   cv: CV
+  shouldUnbookmark?: boolean
 }
 
 interface BookmarkProjectRequest extends NextApiRequest {
   body: BookmarkProjectData
 }
 
 const handler = async (req: BookmarkProjectRequest, res: NextApiResponse) => {
```

Comment:
**issue** one thing with this... as it is currently, can anyone bookmark a project on behalf of anyone else?

I'm assuming that's an issue but might be missing something. lmk

If it is, i guess we'll need some form of API auth... 😓

Reply:
yes, this is true. the idea is to smoke test it for now, then add a session to browsing the website via signing a message in a future PR. this is what i plan to do next.